### PR TITLE
Applied formatting to custom dc docs

### PR DESCRIPTION
**Pull request by dwnoble**
Going through the custom DC quick start, I was running into errors when copying and pasting some of the docker commands due to some trailing whitespace at the end of the line

Example:
```
<pre>
docker run -it \  
-p 8080:8080 \  
-e DEBUG=true \  
--env-file $PWD/custom_dc/sqlite_env.list \  
-v $PWD/custom_dc/<var>CUSTOM_DATA_DIRECTORY</var>:/userdata \  
[-v $PWD/custom_dc/<var>CUSTOM_DATA_DIRECTORY</var>/datacommons:/sqlite]  
gcr.io/datcom-ci/datacommons-website-compose:stable  
</pre>
```

The trailing whitespace after the `\` character causes an error when pasting into a bash shell.

Fix: Applied vscode "prettier" formatting to custom dc markdown files, which, among other formatting fixes, removes extra whitespace before newlines

Steps:
(1) install prettier extension in vs code (open "extensions" tab, search "prettier", install)
(2) enable "format on save option" in vs code preferences (command+shift+p -> type "Preferences: Open User Settings" -> search "Format on Save" -> enable)

### CODE REVIEW

File: custom_dc/build_repo.md [hunk 1]
```diff
@@ -9,13 +9,13 @@ parent: Custom Data Commons
...
--  `gcr.io/datcom-ci/datacommons-website-compose:latest`. This is the latest built version that is running in production.
+- `gcr.io/datcom-ci/datacommons-website-compose:stable`. This is a tested, stable version but may be several weeks old.
+- `gcr.io/datcom-ci/datacommons-website-compose:latest`. This is the latest built version that is running in production.
 
 If you want to pick up the latest prebuilt version, run the following command:
 
-```shell  
-docker pull gcr.io/gcr.io/datcom-ci/datacommons-website-compose:latest  
+```shell
+docker pull gcr.io/gcr.io/datcom-ci/datacommons-website-compose:latest
```
Comment (kmoscoe) [inline custom_dc/build_repo.md:18]: Can you do me one favor? I also found an error here, which

Comment (kmoscoe) [inline custom_dc/build_repo.md:18]:
```suggestion
docker pull gcr.io/datcom-ci/datacommons-website-compose:latest
```

**Review by kmoscoe — APPROVED** [review]
Thank you so much for doing this!!  I will install that plugin and start using right away.

File: custom_dc/build_repo.md (hunk 1 above)
Comment (dwnoble) [reply custom_dc/build_repo.md]: Added!

**Pull request merged**